radio/src/gui/480x272/widget.h

```diff
@@ -106,6 +106,7 @@ class WidgetFactory
       if (options) {
         int i = 0;
         for (const ZoneOption * option = options; option->name; option++) {
+          TRACE("WidgetFactory::initPersistentData() setting option '%s'", option->name);
           // TODO compiler bug? The CPU freezes ... persistentData->options[i++] = option->deflt;
           memcpy(&persistentData->options[i++], &option->deflt, sizeof(ZoneOptionValue));
```

Comment:
Without `maxOptions` parameter for `createOptionsArray()` and with too many options we had a buffer overflow here.